frontend/packages/dev-console/src/components/topology/__tests__/topology-page.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
```

Comment:
can we rename the file to `TopologyPage.spec.tsx`

Reply:
Done